app/src/main/java/com/jerboa/ui/components/settings/block/BlocksScreen.kt

```diff
@@ -86,7 +96,10 @@ fun BlocksScreen(
                     is ApiState.Failure -> ApiErrorText(res.msg)
                     is ApiState.Holder -> {
                         res.data.my_user?.let {
-                            BlockList(it)
+                            BlockList(
+                                userInfo = it,
+                                scope = rememberCoroutineScope()
```

Comment:
What do you gain from passing it here Instead of just defining it in the composable?

Reply:
I think I gain nothing. I copied the UserTabs from the PersonProfileScreen.kt, I saw that it expects the coroutine scope as an argument, but I removed it from the BlockList arguments, thanks.